src/data-mgt/node/config/constants.js

```diff
@@ -2,16 +2,22 @@ const devConfig = {
   MONGO_URI: "mongodb://localhost/",
   JWT_SECRET: process.env.JWT_SECRET,
   DB_NAME: process.env.MONGO_DEV,
+  REDIS_SERVER: process.env.REDIS_SERVER_DEV,
+  REDIS_PORT: process.env.REDIS_PORT,
 };
 const stageConfig = {
   MONGO_URI: process.env.MONGO_GCE_URI,
   JWT_SECRET: process.env.JWT_SECRET,
   DB_NAME: process.env.MONGO_STAGE,
+  REDIS_SERVER: process.env.REDIS_SERVER,
+  REDIS_PORT: process.env.REDIS_PORT,
 };
 const prodConfig = {
   MONGO_URI: process.env.MONGO_GCE_URI,
   JWT_SECRET: process.env.JWT_SECRET,
   DB_NAME: process.env.MONGO_PROD,
+  REDIS_SERVER: process.env.REDIS_SERVER,
```

Comment:
The updated .env can be shared in our private Slack channel. Good idea? Happy to hear your thoughts.

Reply:
Alright... sounds good. Let me share it in the `dev-kampala` private channel